indica/converters/line_of_sight.py

```diff
@@ -49,9 +49,45 @@ class LineOfSightTransform(CoordinateTransform):
     passes
         Number of passes across the plasma (e.g. interferometer
         with corner cube has passes=2)
-
+    beamlets
+        Number of beamlets in the LOS spot cross-section.
+        Currently only works with quadratic sequence (e.g.
+        1^2 = 1, 2^2 = 4, 3^2 = 9, 4^2 = 16, etc...)
+    spot_width
+        Width of the LOS spot in (m).
+    spot_shape
+        Shape of the spot. e.g. "round" or "square"
+    div_width
+        Divergence angle of the width of the spot in (radians).
+    focal_length
+        Focal length of the LOS in (m).
+
+    Examples:
+        Pinhole Cameras
+            origin - position vector of the detector
+            direction - direction vector from detector to the
+                pinhole
+            spot_shape - the spot is defined as the detector
+                cross-section, typically rectangular
+            spot_width - width of the detector
+            focal_length - distance from detector to the centre
+                of the pinhole
+        Spectrometers
+            origin - position vector at a reference position
+                beyond the optics e.g. the front of a lens or
+                mirror position
+            direction - direction vector of the LOS
+            spot_shape - the spot is defined as the cross-section
+                of the LOS, typically round
+            spot_width - width of the spot
+            focal_length - 'effective' distance from a virtual
```

Comment:
Is it worth adding a few extra details, e.g. "'effective' distance from the **spot** to a virtual focal point to define the cone **shape**, typically a negative value **(behind the lens)**" ?